avm/res/document-db/database-account/version.json

```
@@ -1,6 +1,6 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.8",
+    "version": "0.9",
```

```suggestion
    "version": "0.8",
```

This doesn't need a major version bump